This patch makes Python support the vendor-packages directory.
As it is Solaris-specific, it is not suitable for upstream.

--- /dev/null   Sat Feb 12 00:21:26 2011
+++ Python-3.4.0/Lib/site-packages/vendor-packages.pth  Sat Feb 12 00:47:05 2011
@@ -0,0 +1,1 @@
+import site; site.addsitedir('/usr/lib/python3.5/vendor-packages')

